spec/rails_helper.rb

```diff
@@ -74,10 +74,10 @@
 end
 
 VCR.configure do |config|
+  # config.allow_http_connections_when_no_cassette = true
```

Comment:
❓ Should we remove this? I've never used VCR, and would love a quick sit down on how this worked and what challenges you ran into @markcyen !